csharp-basics/exercises/ClassesAndObjects/Exercise3/FuelGauge.cs

```diff
@@ -21,6 +21,27 @@ public void AddFuel()
             }
         }
 
+        public bool IsFullOrEmpty()
```

Comment:
IsFullOrEmpty - true/ yes. Ko tev tas pasaka? Vai nu isFull vai isEmty. Te nav saprotams, ko šis bool pasaka